pages/en/lb3/Polymorphic-relations.md

```diff
@@ -45,7 +45,34 @@ The examples below use three example models: Picture, Author, and Reader, where
 
 ## HasMany polymorphic relations
 
-The usual options apply, for example: `as: 'photos'` to specify a different relation name/accessor.
+Take the following scenario as example:
+
+> A Picture model belongs to either an Author model or a Reader model 
+
+A hasMany polymorphic relation means: there are two properties created in model Picture, one serves as a discriminator, which tells which model the picture actually belongs to, and the other one servers as a foreignKey, which is the id(or primaryKey) value of either the author or reader.
+
+This section explains how to define a hasMany polymorphic relation in modelFrom(Author or Reader in example). You need to also define the corresponding belongsTo relation described in the next section "BelongsTo polymorphic relations" to establish the relation completely.
```

Comment:
This part is redundant: `This section explains how to define a hasMany polymorphic relation`

"If you intend to create a two-way representation of a one-to-many relationship, you will also need to create the `belongsTo` relation on the Model that is the "many" within that relationship. In this example, the Picture model will have two `belongsTo` relationships that reference the Author and Reader models by their primary keys."